Assets/Tests/koob_Assets/Scripts/EnemyController.cs

```diff
@@ -0,0 +1,91 @@
+using System.Collections;
+using System.Collections.Generic;
+using UnityEngine;
+
+public class EnemyController : MonoBehaviour
+{
+    [SerializeField] int hp = 0;
+    [SerializeField] string enemyName = "";
+    [SerializeField] string[] skillName;
```

Comment:
ここら辺の配列も複数形にした方が良いですね。
指摘漏れでした。すみません。
```suggestion
    [SerializeField] string[] skillNames;
```